apps/client/src/app/components/not-found/not-found.component.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+
+  <title>404 HTML Template by Colorlib</title>
+
+  <!-- Google font -->
+  <link href="https://fonts.googleapis.com/css?family=Kanit:200" rel="stylesheet">
+
+  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+  <!--[if lt IE 9]>
+  <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
+  <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+  <![endif]-->
+
+</head>
+
+<body>
+
+<div id="notfound">
+  <div class="notfound">
+    <div class="notfound-404">
+      <h1>404</h1>
+    </div>
+    <h2>Oops! Nothing was found</h2>
+    <p>The page you are looking for might have been removed, had its name changed or is temporarily unavailable.</p>
+    <button mat-raised-button>HOME</button>
```

Comment:
Could you implement redirection to "home" page by clicking this button?

Reply:
I've updated the PR as you requested. Please check.